src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task07.java

```diff
@@ -38,6 +38,12 @@ public static void main(String[] args) {
     static boolean isLeapYear(int year) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return false;
+        boolean leapYearResult;
+            if ((year % 400 == 0) || (year % 4 == 0) && ((year % 100 != 0)) ) {
+                leapYearResult = true;
+            } else {
+                leapYearResult = false;
+            }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"